libraries/FusionData/FusionData.h

```diff
@@ -1,12 +1,13 @@
+#ifndef FUSIONDATA
+#define FUSIONDATA
 // FusionData.h - header included for all functions to fuse the 
 // IMU sensor data, speed data and GPS data
 
-
-#define HEADING_PRECISION 1000
+#define HEADING_PRECISION_DEG 1000
 // Hard coded values
-#define COMPASS_ERROR ((double) 2)
-#define DISTANCE_ERROR ((double) 0.04)
-#define GPS_ERROR ((long) 4236)
+#define COMPASS_ERROR_DEG ((double) 2)
+#define DISTANCE_ERROR_MM ((double) 0.04)
+#define GPS_ERROR_DEG ((long) 4236)
```

Comment:
Unless these constants are not used anywhere else, it would be better to make name changes in a separate PR, and update the files that use them in the same PR.  (That's another Tradition:  He who makes the change, fixes the dependencies. :-D  A.k.a. Don't break the build!)